tjeaneric/Flashcards-API
# Finished CRUD operations on Flashcards

#### What does this PR do?
* CRUD(Create, Read, Update, Delete) operations on flashcards


#### Description of Task to be completed?
* Users can perform Create, Read, Update and Delete operations on flashcards.

#### How should this be manually tested?
* clone the repo using SSH: `git clone git@github.com:tjeaneric/Flashcards-API.git`
                          or    HTTPS: `git clone https://github.com/tjeaneric/Flashcards-API.git`
* checkout to this branch by running `git checkout CRUD-operations-for-flashcards`
* run `npm install`
* check env.example file and add required environment variables to your .env file
* Do migrations with `npx prisma migrate dev --name "init"`
* run `npm run dev`
* got to your localhost:{port}, you will see your apollo server studio and perform CRUD operations using Mutations and Queries

#### Any background context you want to provide?
No.

#### Questions:
No.
